src/openrct2/Game.cpp

```diff
@@ -573,6 +573,7 @@ void game_load_init()
     reset_sprite_spatial_index();
     reset_all_sprite_quadrant_placements();
     scenery_set_default_placement_configuration();
+    map_calc_highest_tile_height(true);
```

Comment:
On game load, we check the whole map to make sure the viewport doesn't have forced down for tall parks where the game was saved panned all the way to the top.